.github/workflows/test-command.yml

```diff
@@ -124,8 +124,8 @@ jobs:
           SOURCE_SQUARE_CREDS: ${{ secrets.SOURCE_SQUARE_CREDS }}
           SOURCE_MARKETO_SINGER_INTEGRATION_TEST_CONFIG: ${{ secrets.SOURCE_MARKETO_SINGER_INTEGRATION_TEST_CONFIG }}
           SOURCE_RECURLY_INTEGRATION_TEST_CREDS: ${{ secrets.SOURCE_RECURLY_INTEGRATION_TEST_CREDS }}
-          STRIPE_INTEGRATION_TEST_CREDS: ${{ secrets.STRIPE_INTEGRATION_TEST_CREDS }}
-          STRIPE_INTEGRATION_CONNECTED_ACCOUNT_TEST_CREDS: ${{ secrets.STRIPE_INTEGRATION_CONNECTED_ACCOUNT_TEST_CREDS }}
+          SOURCE_STRIPE_CREDS: ${{ secrets.SOURCE_STRIPE_CREDS }}
+          # STRIPE_INTEGRATION_CONNECTED_ACCOUNT_TEST_CREDS: ${{ secrets.STRIPE_INTEGRATION_CONNECTED_ACCOUNT_TEST_CREDS }}
```

Comment:
this should not be commented out because some tests should depend on it

Reply:
ok